Lib/test/test_http_cookies.py

```diff
@@ -59,6 +59,47 @@ def test_basic(self):
             for k, v in sorted(case['dict'].items()):
                 self.assertEqual(C[k].value, v)
 
+    def test_obsolete_rfc850_date_format(self):
+        # Test cases with different days and dates in obsolete RFC 850 format
+        test_cases = [
+            {
+                'data': 'key=value; expires=Saturday, 01-Jan-83 00:00:00 EST',
```

Comment:
Ah I think I know why the tests are failing. The examples seem to use EST but the regex seems to expect GMT only. Should we allow more timezones?

Reply:
https://www.rfc-editor.org/rfc/rfc9110.html#section-5.6.7-12

Maybe GMT only is right. I will fix the test cases.